e2e-tests/global-setup.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { chromium, test as baseTest } from '@playwright/test';
+import { checkEnv } from './helpers/general.js';
+
+import { LoginPage } from './admin/pages/login.js';
+
+async function globalSetup() {
+  await checkEnv([
+    'BOUNDARY_ADDR',
+    'E2E_PASSWORD_ADMIN_LOGIN_NAME',
+    'E2E_PASSWORD_ADMIN_PASSWORD',
+    'E2E_PASSWORD_AUTH_METHOD_ID',
+  ]);
+
+  // Log in and save the authenticated state to reuse in tests
```

Comment:
Sure, I can extract that out but that's basically the entire `globalSetup` haha

Reply:
I am fine if you just add a comment saying in case the `globalSetup` grows, we can extract the login part.

I like the suggested separation of concerns because it's a very specific action, log in and take the token. Which now seems most, but not all, part `globalSetup`, to me is cleaner to understand. As mentioned on the comment, I will not block the PR for it 😉